### PR TITLE
Added "no-report-server" option

### PR DESCRIPTION
**Pull request by narwold**
I ran into an issue where I was trying to run your code in CI/CD, and it stalled just after starting the report server to show the report in the browser. This also happened locally, where it stalled for a while at this point sometimes, but not as bad as in CI/CD. I'm not in need of the HTML report, so this PR adds an option to disable it for a given invocation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "deadfile",
+  "name": "@foundation-lib/deadfile",
```
Comment (M-Izadmehr) [inline package.json:2]: Could you please change this back to deadfile?

File: src/bin.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ simple:                $0 ./src/index.js
 multiple entry:        $0 ./src/index.js ./src/entry2.js
 with custom directory: $0 ./src/index.js --dir /path/to/other/folder
 with  exclude:         $0 ./src/index.js --exclude tests  utils/webpack
+without report server: $0 ./src/index.js --no-report-server utils/webpack
```
Comment (M-Izadmehr) [inline src/bin.js:18]: we also need to add the required changes for this option in the `src/cli/options.js`

**Comment by M-Izadmehr** [comment]
Hi @narwold,
Thanks for taking the time and helping with this project. Sorry that I wasn't active in github for the past few months.
Please let me know if you are still interested in finishing this PR. If you are busy, no worries!
I can add the required changes on top of you PR and merge it:)

Thanks again!

File: package.json (hunk 1 above)
Comment (narwold) [reply package.json:2]: Yep; sorry; should have done that right away after publishing to my private scope

File: src/bin.js (hunk 2 above)
Comment (narwold) [reply src/bin.js:18]: Done

**Comment by narwold** [comment]
> Hi @narwold,
> Thanks for taking the time and helping with this project. Sorry that I wasn't active in github for the past few months.
> Please let me know if you are still interested in finishing this PR. If you are busy, no worries!
> I can add the required changes on top of you PR and merge it:)
> 
> Thanks again!

No worries at all. I know life gets busy. Thanks for taking a look at this!

**Comment by luke-lacroix-healthy** [comment]
I didn't see this PR before I opened my own (#64). It does, essentially, the same thing.

**Pull request closed without merging**